packages/oscal-react-library/src/components/OSCALAnchorLinkHeader.tsx

```diff
@@ -12,7 +12,16 @@ const AnchorLinkIcon = styled(LinkIcon)(({ theme }) => ({
   color: theme.palette.grey[500],
 }));
 
-export default function OSCALAnchorLinkHeader(props) {
+export interface AnchorLinkProps {
+  urlFragment?: string;
+}
+
+export interface OSCALAnchorLinkHeaderProps {
+  children: any;
+  value?: any;
```

Comment:
Would love to see someone suffer the pain of renaming this to a _way_ more meaningful name once we know what it actually does lol

Reply:
`conformLinkIdText` is the reason that this function is hard to assign types to (I think). We are allowed to put pretty much anything as an anchor link, which I feel like is a bit weird. However I am unsure how to fix that helper function without rewriting a lot of how we do anchor links.

Reply:
```suggestion
  name?: string;
```

`name` is probably more fitting. This is just an optional `string` that is provided for the id of the anchor link and fragment.